jetpack.php

```diff
@@ -175,29 +154,9 @@ function jetpack_admin_missing_autoloader() { ?>
 	return;
 }
 
-/**
- * Load all the packages.
- *
- * We want to fail gracefully if `composer install` has not been executed yet, so we are checking for the autoloader.
- * If the autoloader is not present, let's log the failure, pause Jetpack, and display a nice admin notice.
- */
-$jetpack_autoloader = JETPACK__PLUGIN_DIR . 'vendor/autoload.php';
-if ( is_readable( $jetpack_autoloader ) ) {
-	require $jetpack_autoloader;
-} else {
-	if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
-		error_log(
-			sprintf(
-				/* translators: Placeholder is a link to a support document. */
-				__( 'Your installation of Jetpack is incomplete. If you installed Jetpack from GitHub, please refer to <a href="%1$s" target="_blank" rel="noopener noreferrer">this document</a> to set up your development environment.', 'jetpack' ),
-				esc_url( 'https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md' )
-			)
-		);
-	}
-	add_action( 'admin_notices', 'jetpack_admin_missing_autoloader' );
-	return;
-}
-
+require_once( JETPACK__PLUGIN_DIR . 'packages/autoloader-loader/src/AutoloaderLoader.php' );
```

Comment:
I see that this is still in progress, but let me just note one thing here. We shouldn't be using the `packages` folder at all - we should be including things from the `vendor` folder because it's the one folder that theoretically contains all code from dependencies in the production build.